requirements/app.txt

```diff
@@ -2,8 +2,8 @@ galaxy.plugin.api==0.69
 
 # steam_network backend
 protobuf==3.19.4
-vdf==3.2
-pyobjc-framework-CoreServices==5.1.2; sys_platform == 'darwin'
-websockets==8.1
-dataclasses-json==0.4.2
+vdf==3.4
+pyobjc-framework-CoreServices==5.1.2; sys_platform == 'darwin'  # this one could use an update to 9.1.1 I guess; according to docu it's tested with 3.7 regularly
```

Comment:
I can try to get our Mac dev to try that latest version. The other potential issue is the newer versions might have some Mac issues we don't know about. I doubt it but i know from my own testing that pip tools 5.1.2 was broken on Mac (which we inherited from the old code). It required 5.4.0. A newer version should still work there but it's something we'll need to confirm with all of these updated versions.

Reply:
https://pyobjc.readthedocs.io/en/latest/changelog.html#backward-incompatible-changes
I have no idea where we use this but this might be an issue. Idk if we abused the optional argument rule before they nixed it in 8.0

Reply:
I'm wondering why we even need to include this specific dependency. We don't use any Objc bindings anywhere so this appears to me to be a band aid to fix another dependency on MacOS.

Reply:
Uri Scheme Handler uses it. i have no idea why/what it does though.  See https://github.com/ABaumher/galaxy-integration-steam/blob/4a84a25f98356a1759ac464032010ab05685db96/src/uri_scheme_handler.py#L28-L36